x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_logic.test.ts

```diff
@@ -29,6 +29,15 @@ describe('AnalyticsLogic', () => {
     dataLoading: true,
     analyticsUnavailable: false,
     allTags: [],
+    totalQueries: 0,
+    totalQueriesNoResults: 0,
+    totalClicks: 0,
+    totalQueriesForQuery: 0,
+    queriesPerDay: [],
+    queriesNoResultsPerDay: [],
+    clicksPerDay: [],
+    queriesPerDayForQuery: [],
+    startDate: '',
```

Comment:
[Definitely not proposing any changes now, just thinking out loud] I honestly don't know if I'm a super huge fan of storing just a ton of raw data from the server-side APIs as a billion top-level variables. It feels kinda messy and strange especially since we're not really _doing_ anything with this state.

I'm not 100% sure but I think I slightly prefer how EngineLogic handles it, which is storing the entire response blob in a single `engine` var... so maybe we'd just store response data in a generic `data` or `response` obj. In theory Typescript should take care of any data mismatches for us, and all we have to do is make sure our types are consistent/up to date with what the API sends back.

I dunno, what do y'all think? 🤷

Reply:
I think if we don't expect these values to be updated by the logic individually, it makes sense to bundle them into some sort of `analytics` property

Reply:
Cool, maybe something to think about for our next migration :trollface: